app/Http/Controllers/AboutController.php

```diff
@@ -4,18 +4,44 @@
 
 use App\Models\Contributor;
 use Illuminate\Contracts\View\View;
+use Illuminate\Support\Facades\Cache;
+use Illuminate\Support\Facades\File;
+use JsonException;
 
 final readonly class AboutController
 {
+    private const DAY = 60 * 60 * 24;
+
     public function __invoke(): View
     {
-        $contributors = array_map(
-            fn (array $item) => new Contributor(...$item),
-            json_decode(file_get_contents(__DIR__.'/../../../contributors.json') ?: '{}', true)['contributors'] ?? [],
-        );
-
-        return view('about', [
-            'contributors' => $contributors,
-        ]);
+        $contributors = Cache::remember('contributors', self::DAY, $this->getContributors(...));
+
+        shuffle($contributors);
+
+        return view('about', compact('contributors'));
+    }
+
+    /**
+     * @return Contributor[]
+     *
+     * @throws JsonException
+     */
+    private function getContributors(): array
+    {
+        /**
+         * @var array{
+         *     contributors: array<int, array{
+         *         id: int,
+         *         name: string,
+         *         url: string,
+         *         contributions: array<int, string>,
+         *     }>
+         * } $contributors
+         */
+        $contributors = File::json(__DIR__.'/../../../contributors.json', JSON_THROW_ON_ERROR);
```

Comment:
```suggestion
        $contributors = File::json(base_path('contributors.json'), JSON_THROW_ON_ERROR);
```

Reply:
I forgot about `base_path()`, was looking for `root_path()` but couldn't find it 🤦